float64 steering_angle
float64 steering_angle_rate
float64 angular_velocity